titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/event/JobModelReconcilerEvent.java

```diff
@@ -18,22 +18,23 @@
 
 import java.util.Optional;
 
+import com.netflix.titus.api.jobmanager.model.CallMetadata;
 import com.netflix.titus.api.jobmanager.model.job.Job;
 import com.netflix.titus.common.framework.reconciler.EntityHolder;
 import com.netflix.titus.common.framework.reconciler.ModelActionHolder;
 import com.netflix.titus.master.jobmanager.service.common.action.TitusChangeAction;
 
 public abstract class JobModelReconcilerEvent extends JobManagerReconcilerEvent {
 
-    protected JobModelReconcilerEvent(Job<?> job, String transactionId) {
-        super(job, transactionId);
+    protected JobModelReconcilerEvent(Job<?> job, String transactionId, CallMetadata callMetadata) {
+        super(job, transactionId, callMetadata);
     }
 
     public static class JobNewModelReconcilerEvent extends JobModelReconcilerEvent {
         private final EntityHolder newRoot;
 
         public JobNewModelReconcilerEvent(EntityHolder newRoot) {
-            super(newRoot.getEntity(), "-1");
+            super(newRoot.getEntity(), "-1", CallMetadata.newBuilder().withCallerId("reconciler event").build());
```

Comment:
This is an initial job create event, which will be always initiated by a user. We could set here the same `CallMetadata` as for the job create. It would have to be passed directly.